scripts/deploy.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const { printLog, command, runMain } = require('./utils')
+
+const ONE_MINUTE_IN_SECOND = 60
+const ONE_HOUR_IN_SECOND = 60 * ONE_MINUTE_IN_SECOND
+const AWS_CONFIG = {
+  prod: {
+    accountId: 464622532012,
+    bucketName: 'browser-agent-artifacts-prod',
+    distributionId: 'EGB08BYCT1DD9',
+  },
+  staging: {
+    accountId: 727006795293,
+    bucketName: 'browser-agent-artifacts-staging',
+    distributionId: 'E2FP11ZSCFD3EU',
+  },
+}
+/**
+ * Deploy SDK files to CDN
+ * Usage:
+ * node deploy.js staging|prod staging|canary|vXXX
+ */
+const env = process.argv[2]
+const version = process.argv[3]
+
+const browserCache = version === 'staging' || version === 'canary' ? 15 * ONE_MINUTE_IN_SECOND : 4 * ONE_HOUR_IN_SECOND
+const cacheControl = `max-age=${browserCache}, s-maxage=60`
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏this could be moved in `uploadToS3`